Feature 948 implementation of ramping parameters

stage1407 opened the pull request:
`Closes #948`
#948 was about adding ramping and two other concepts.

stage1407 commented:
Ok my pushes produced a lot of inconsistencies.
I don't know whether i can fix it until the release, but I'll try my best.

stage1407 commented:
I think now anyone can review my changes.

l-emele commented:
> I think now someone can review my changes.

It is good practise to assign those people involved in the issue discussion as reviewers.

I will start reviewing now.

l-emele reviewed (changes requested):
I noticed the following things.
* I don't understand, why with this PR also the `industrial material` classes should be added, these are already included in the OEO. Maybe try merging the `dev` branch into your feature branch.
* Changes have to be documented not only in term tracker items, but also in the file `CHANGELOG.md`.
* Axiom `'cold start' 'has process attribute' some 'start-up speed'` needs to be added, maybe also the inverse `'start-up speed' 'process attribute of' some 'cold start'`.

Further, I am unsure about the axiom `ramping and ('happens during' some 'cold start')`. Usually, we do axioms like this only for equivalent classes.

stage1407 commented:
Ok I have now modified my implementation.

Because of `industrial materials` I think that I had deleted it by mistake.
When I realized that some things from the ontology were missing I replaced my physical.omn with the one stored in the DEV branch.

I think that `industrial materials` are noted as added classes by Github.

stage1407 commented:
@l-emele I think we can merge now unless you have further comments.

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -8046,6 +8049,44 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1082",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Ramping is a process attribute that describes the change in power of an energy transformation unit or an energy converting component per time step.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/948
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1126",
+        rdfs:label "ramping"@de
```
```suggestion
        rdfs:label "ramping"@en
```

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -8046,6 +8049,44 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1082",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "Start-up speed is ramping during a cold start.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/948
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1126",
+        rdfs:label "start-up speed"@de
```
```suggestion
        rdfs:label "start-up speed"@en
```

l-emele commented on src/ontology/edits/oeo-physical.omn:
```diff
@@ -8046,6 +8049,44 @@ pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1082",
+        <http://purl.obolibrary.org/obo/IAO_0000115> "A cold start is a process in which an energy transformation unit is started-up after a period of inactivity.",
+        <http://purl.obolibrary.org/obo/IAO_0000233> "issue: https://github.com/OpenEnergyPlatform/ontology/issues/948
+pull request: https://github.com/OpenEnergyPlatform/ontology/pull/1126",
+        rdfs:label "cold start"@de
```
```suggestion
        rdfs:label "cold start"@en
```

It is English language, so it should be either `@en` or nothing at all.

l-emele reviewed (approved):
I did some small changes. Now everything looks fine and I thus approve this PR.
